modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported log data output types
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 provides the following output types and protocols for sending log data to target log collectors.
+
+Red Hat tests each of the combinations shown in the following table. However, you should be able to send log data to a wider range target log collectors that ingest these protocols.
+
+[options="header"]
+|====
+| Output types   | Protocols          | Target log collector
+
+| fluentdForward | fluent forward v1  | fluentd 1.7.4
+
+| elasticsearch  | elasticsearch v?	  | Elasticsearch 6.8.1
+
+| syslog         | RFC-3164, RFC-5424 | <What syslog server & version?>
```

Comment:
@vimalk78  Would you provide the target Syslog server and version?

Reply:
In our tests we use quay.io/openshift/origin-logging-rsyslog:latest image. 
i dont think we officially support any particular syslog server

Reply:
for testing we use `rsyslog 8.37.0-9.el7`